openapi-v1.yaml

```diff
@@ -6343,24 +6242,67 @@ paths:
         type: string
         in: path
         required: true
-        description: The name of the group
-    post:
-      description: Creates a new, empty group in the namespace.
-      operationId: createGroup
-      tags:
-        - groups
-      parameters:
-        - in: body
-          name: GroupCreate
-          schema:
-            $ref: '#/definitions/GroupCreate'
-      responses:
-        204:
-          description: group created successfully
-        default:
-          description: error response
-          schema:
-            $ref: "#/definitions/Error"
+        description: The name or id of the group
+      - name: page
+        in: query
+        description: pagination offset
+        type: integer
+        required: false
+      - name: per_page
+        in: query
+        description: pagination limit
+        type: integer
+        required: false
+      - name: search
+        in: query
+        description: search string that will look at name, namespace or description fields
+        required: false
+        type: string
+      - name: orderby
+        in: query
+        description: sort by which field valid values include last_accessed, size, name
+        required: false
+        type: string
+      - name: tag
+        in: query
+        description: tag to search for, more than one can be included
+        required: false
+        type: array
+        collectionFormat: multi
+        items:
+          type: string
+      - name: exclude_tag
+        in: query
+        description: tags to exclude matching array in results, more than one can be included
+        required: false
+        type: array
+        collectionFormat: multi
+        items:
+          type: string
+      - name: file_type
+        in: query
+        description: file_type to search for, more than one can be included
+        required: false
+        type: array
+        collectionFormat: multi
+        items:
+          type: string
+      - name: exclude_file_type
+        in: query
+        description: file_type to exclude matching array in results, more than one can be included
+        required: false
+        type: array
+        collectionFormat: multi
+        items:
+          type: string
+      - name: file_property
+        in: query
+        description: file_property key-value pair (comma separated, i.e. key,value) to search for, more than one can be included
+        required: false
+        type: array
+        collectionFormat: multi
+        items:
+          type: string
     get:
       description: Returns the contents, assets and subgroups, of the group
       operationId: listGroup
```

Comment:
getGroupContent or listGroupContent would be better (for auto-generated clients)

Reply:
👍